template-compose/app/src/main/java/co/nimblehq/template/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -1,24 +1,39 @@
 package co.nimblehq.template.compose.ui.screens.home
 
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.*
 import androidx.compose.material.Text
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.hilt.navigation.compose.hiltViewModel
+import co.nimblehq.template.compose.R
 import co.nimblehq.template.compose.model.UiModel
+import co.nimblehq.template.compose.ui.AppDestination
+import co.nimblehq.template.compose.ui.theme.ComposeTheme
 import co.nimblehq.template.compose.ui.theme.Dimension.SpacingNormal
-import co.nimblehq.template.compose.ui.theme.Theme
 import timber.log.Timber
 
-@ExperimentalComposeUiApi
 @Composable
 fun HomeScreen(
+    viewModel: HomeViewModel = hiltViewModel(),
+    navigator: (destination: AppDestination) -> Unit
+) {
+    val uiModels: List<UiModel> by viewModel.uiModels.collectAsState()
+
+    HomeScreenContent(
+        title = stringResource(id = R.string.app_name),
+        uiModels = uiModels
+    )
+}
+
+@Composable
+private fun HomeScreenContent(
     title: String,
     uiModels: List<UiModel>
 ) {
-    Theme {
+    Column {
```

Comment:
@luongvo It's still not in the center of the screen, I think we need to define `modifier` and `verticalArrangement`: 

```
    Column(
        modifier = Modifier.fillMaxSize(),
        verticalArrangement = Arrangement.Center
    ) {
        Text(
            text = title,
            textAlign = TextAlign.Center,
            modifier = Modifier
                .fillMaxWidth()
                .padding(all = SpacingNormal),
        )
    }
```
My last concern for the rest all good 🚀

Reply:
@Wadeewee can you make a suggestion on actual code block instead?

Reply:
@luongvo 👇 Here's form my side
**Before:**
![Screen Shot 2565-12-28 at 11 04 40](https://user-images.githubusercontent.com/28002315/209755697-d2205b05-29a7-4a67-a24a-0a3f2106b1d3.png)
**After:**
![Screen Shot 2565-12-28 at 11 05 28](https://user-images.githubusercontent.com/28002315/209755732-c90908a3-de52-47a1-ac8f-13a028465173.png)

Reply:
@Wadeewee I mean `Github suggestion` via comment. I can just apply it without opening the project in my AS again 😫 

<img width="808" alt="image" src="https://user-images.githubusercontent.com/16315358/209756345-47bb0978-7928-4808-939b-8e07d39fb1ae.png">

Reply:
Your code looks good 👍

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/372/commits/27d967ee14c09dccbc4787ea2269d31cfad43d24